web_dialog_size/static/src/xml/web_dialog_size.xml

```diff
@@ -5,12 +5,12 @@
             <i class="fa fa-close" />
         </t>
         <t t-jquery="button.close" t-operation="before">
-            <button type="button" class="dialog_button_extend close"><i
-                    class="fa fa-expand"
-                /></button>
-            <button type="button" class="dialog_button_restore close"><i
-                    class="fa fa-compress"
-                /></button>
+            <button type="button" class="dialog_button_extend close">
+                <i class="fa fa-expand" />
+            </button>
+            <button type="button" class="dialog_button_restore close">
+                <i class="fa fa-compress" />
+            </button>
```

Comment:
I wonder if in cases like this whitespace actually makes a difference... :thinking:

Reply:
Why it should be different?

Reply:
It could render a whitespace surrounding the icon that could break the layout.

Reply:
I don't think so, ad the space is only being added inside `<i>` tag, or am I missing something?

Reply:
I think such white space is not significant *by default*: https://www.w3.org/TR/css-text-3/#white-space-phase-2. But there are css properties that influence that (e.g. `whitespace: pre`) so in theory this could break some files. Such breakage is very unlikely in the OCA codebase IMO.

Reply:
No, well, you know sometimes inside an `<a>` if there are whitespaces they form an ugly link with spaces underlined.

It doesn't seem to be the case here anyways:

![2020-03-02_10-38](https://user-images.githubusercontent.com/973709/75669003-1cf3fa80-5c72-11ea-9e8e-f016924d4ef4.png)

(pic from runbot).